tests/unit/s2n_client_hello_test.c

```diff
@@ -1953,6 +1959,110 @@ int main(int argc, char **argv)
         };
     };
 
+    /* Test: Client Hello is slightly less than 64KB */
+    {
```

Comment:
These tests require a lot of setup that should be the same. Can they share any setup, to better highlight the differences?